.github/workflows/get-previous-release/action.yml

```diff
@@ -13,8 +13,13 @@ runs:
       run: |
         # If the channel is either "cloud" or "beta", fetch all Git tags matching the channel pattern,
         # excluding any "-rc" tags, and store them in the 'tags' variable. e.g. "refs/tags/v3.11.0-cloud1 , refs/tags/v3.11.0-cloud2"
-        if [[ ${{ inputs.CHANNEL }} == "cloud" || ${{ inputs.CHANNEL }} == "beta" ]]; then
-           tags=$(git ls-remote --tags | grep ${{ inputs.CHANNEL }} | grep -v "\-rc" | awk '{print $2}')
+        if [[ ( ${{ inputs.CHANNEL }} == "cloud" || ${{ inputs.CHANNEL }} == "beta" ) && "${{ env.POSTFIX }}" != "-test" ]]; then
```

Comment:
Lets pass postfix as a parameter to the action

Reply:
Also, we don't have to check for the POSTFIX here, it will just be overridden in the next `if` clause